swiftly/cli/context.py

```diff
@@ -38,11 +38,7 @@ def __getattr__(self, name):
             return None
 
     def __repr__(self):
-        result = super(CLIContext, self).__repr__()
-        for item in dir(self):
-            if item[0] != '_' and item != 'copy':
-                result += '\n    %s = %s' % (item, getattr(self, item))
-        return result
```

Comment:
string formatting for a method of an instantiated object in python3 will call __repr__ for that object:

```python

>>> class Foo:
...   def bar():
...     pass
... 
>>> f = Foo()
>>> str(f.bar)
'<bound method Foo.bar of <__main__.Foo object at 0x7f81c09b81d0>>'
```

…infinite recursion. vars() does basically this, so use that instead.